orc8r/cloud/go/services/tenants/client_api.go

```diff
@@ -20,8 +20,9 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"magma/orc8r/cloud/go/services/tenants/protos"
 	merrors "magma/orc8r/lib/go/errors"
-	"magma/orc8r/lib/go/protos"
+	common_protos "magma/orc8r/lib/go/protos"
```

Comment:
One nit: let's not alias this one (so it's just `protos`), and instead alias the the more-specific protos (so e.g. `tenant_protos`)